spec/spec_helper.rb

```diff
@@ -21,6 +21,14 @@
 
   config.filter_run_excluding ruby: ->(v) { !RUBY_VERSION.start_with?(v.to_s) }
 
+  ruby_version = Gem::Version.new(RUBY_VERSION)
+  config.filter_run_excluding ruby_less_than: (lambda do |v|
+    ruby_version >= Gem::Version.new(v)
+  end)
+  config.filter_run_excluding ruby_greater_than: (lambda do |v|
+    ruby_version <= Gem::Version.new(v)
+  end)
```

Comment:
Hmm, this is breaking due to patch versions (2.1.x) is greater than 2.1. I could change greater_than to be greater than or equal, since I think the use case is only 'at least this version' and 'less than this version'. Or, I could add in 'less than', 'less than or equal', 'greater than' and 'greater than or equal' and then just let the spec writer choose which ever is correct.